templates/provider/hmc-templates/files/templates/cluster-api-provider-openstack.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: ProviderTemplate
+metadata:
+  name: cluster-api-provider-openstack-0-0-1
+  annotations:
+    helm.sh/resource-policy: keep
+spec:
```

Comment:
Could you please re-run `make templates-generate`? The Template struct has been changed and should be updated.

Reply:
Done.